tests/PCA.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+import numpy as np
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+import matplotlib
+import matplotlib.pyplot as plt
+
+#defining the function for PCA
+def Compute_PCA(data_to_compute, num_components, show_results=False):
+    """THIS FUNCTOIN TAKES DATAFRAME AS INPUT, CALCULATES THE PRINCIPLE COMPONENTS AND RETURS A DATAFRAME CONTAINING THE PRINCIPAL COMPONETS INSIDE IT"""
```

Comment:
Please modify the docstring so that it looks like the docstrings in our other toolkit functions. Take a look at for example the clipping function. I suggest you copy-paste the docstring of clip for example, and modify the contents according your function.